crypto/ec/curve25519.c

```diff
@@ -763,68 +763,71 @@ static const int64_t kBottom26Bits = 0x3ffffffLL;
 static const int64_t kTop39Bits = 0xfffffffffe000000LL;
 static const int64_t kTop38Bits = 0xfffffffffc000000LL;
 
-static uint64_t load_3(const uint8_t *in) {
-  uint64_t result;
-  result = (uint64_t)in[0];
-  result |= ((uint64_t)in[1]) << 8;
-  result |= ((uint64_t)in[2]) << 16;
-  return result;
+static uint64_t load_3(const uint8_t *in)
+{
+    uint64_t result;
+    result = (uint64_t)in[0];
```

Comment:
Blank line after the declaration.  consider indenting the expression so the cast/in part lines up with the next two lines.

Reply:
> Blank line after the declaration.

Thanks for the reminder. Needs to be fixed globally.